app/emails/templates/request_access.md

```diff
@@ -0,0 +1,15 @@
+Hello,  
+  
+{{ name }} has requested **{{ permission }}** permission.  
+  
+This person does not currently have this permission.  
+  
+To grant this permission, go to [**Teams and people**]({{ frontend_url }}/2/teams) and update their permissions.
```

Comment:
Should we be linking directly to the permissions stage?

Reply:
it might be safer to let them navigate to it.